tests/JWT.Extensions.AspNetCore.Tests/JWT.Extensions.AspNetCore.Tests.csproj

```diff
@@ -2,6 +2,8 @@
 
   <PropertyGroup>
     <TargetFramework>netcoreapp2.0</TargetFramework>
+    <SignAssembly>true</SignAssembly>
```

Comment:
Do we really need to sign tests? I don't think we do. So having the file on /src would skip signing tests.

Reply:
Yeah, that's fair. I don't personally have a need to reference the test assemblies in another project. I'd probably just give them a strong name anyway because it doesn't hurt anything and can only potentially help. But I'm fine either way. What do you think?

Reply:
I don't have a preference really either. But I see the props file already exists under /src and having a hierarchy of them is little more complex than having a single one. So that's the only reason I'd go with signing only under /src.

Reply:
Ah, I didn't see the file there. Yes, that makes sense to me too.

Reply:
Actually, I found a problem with that. The InternalsVisibleTo attribute is being used and it requires the specified assembly to have a strong name (when its own assembly has a strong name). So, I think signing the test assemblies is best.

Reply:
It seems it doesn't like having Directory.Build.props at multiple levels. So, I'll add the property in both the /src and /tests directories.

Reply:
Here's how you can make it respecting the hierarchy of such files: add this line to the top of the nested file:
```xml
  <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.props', '$(MSBuildThisFileDirectory)..\'))" />```